Unify OTLP path parsing/default Logic (#2639)
open-telemetry/opentelemetry-go: CHANGELOG.md

- For tracestate's members, prepend the new element and remove the oldest one, which is over capacity (#2592)
- Add event and link drop counts to the exported data from the `oltptrace` exporter. (#2601)

> - Unify of path parsing functionally in `otlpmetric` and `otlptrace` config. (#2639)

Reviewer: ```suggestion
- Unify path cleaning functionally in the `otlpmetric` and `otlptrace` config. (#2639)
```
